### PR TITLE
[stable/redis Add JSON Schema

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juan@bitnami.com>

#### Is this a new chart

No

**Description of the change**

This PR includes a basic JSON schema for the Redis chart. It doesn't cover all the possible values of the chart but just the ones that, in my opinion, are more likely to be modified.

This is valid for Helm 3 to validate some values and, at the same time, for Kubeapps to render a simple form so the chart containing it is easier to configure and deploy. See https://github.com/kubeapps/kubeapps/blob/master/docs/developer/basic-form-support.md for more information.

This is an example of the file rendered by Kubeapps:

Screenshot 2019-11-28 at 16 33 36

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/redis/values.schema.json
Comment (andresmgot) [inline stable/redis/values.schema.json]: same than https://github.com/bitnami/charts/pull/1679#discussion_r352032175

File: stable/redis/values.schema.json
Comment (andresmgot) [inline stable/redis/values.schema.json]: same issue for the requests

File: stable/redis/values.schema.json
Comment (andresmgot) [inline stable/redis/values.schema.json]: can we give a better description for people that are not familiar with the chart? This seems very specific

**Review by andresmgot — COMMENTED** [review]
Thanks @juan131, see my comments

File: stable/redis/values.schema.json
Comment (juan131) [reply stable/redis/values.schema.json]: Sure

**Review by andresmgot — APPROVED** [review]
Thanks for the changes!

**Comment by davidkarlsen** [comment]
@andresmgot Is this good to go now?

**Comment by andresmgot** [comment]
yes, it is. It's just that I am not a maintainer here so we need the `/lgtm` from @carrodher

**Pull request merged**
